pkg/render/compliance.go

```diff
@@ -124,18 +124,25 @@ func (c *complianceComponent) Objects() []runtime.Object {
 		c.complianceGlobalReportNetworkAccess(),
 		c.complianceGlobalReportPolicyAudit(),
 		c.complianceGlobalReportCISBenchmark(),
+
+		// Even though we do not have a compliance-server, we need a serviceaccount
```

Comment:
This comment is a little misleading I think. "Even though we do not have a compliance-server", this code is common to both managed and management. Maybe it should be "Even if we do not have a compliance-server".